__tests__/server/utils/heapdump.spec.js

```diff
@@ -14,72 +14,79 @@
  * permissions and limitations under the License.
  */
 
-/* eslint import/no-extraneous-dependencies: ["error", {
-  "devDependencies": false,
-  "optionalDependencies": true,
-  "peerDependencies": false
-}] */
+const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
 
 describe('safeRequest', () => {
```

Comment:
huh, seems like a previous copy+paste error
```suggestion
describe('heapdump', () => {
```